README.md

```diff
@@ -187,6 +188,11 @@ git help -g
 git log -S'<a term in the source>'
 ```
 
+## Remove sensitive data from history, after a push
+```sh
+git filter-branch --force --index-filter 'git rm --cached --ignore-unmatch <path-to-your-file>' --prune-empty --tag-name-filter cat -- --all && git push origin --force --all
```

Comment:
`force` push...can you elaborate, please?

Reply:
`force` is there as the command should verify overwrite of existing remote data. see [this post on github help][1].



[1]: https://help.github.com/articles/removing-sensitive-data-from-a-repository/

Reply:
I understand what `force` dose, but we can't avoid it here?

Reply:
the purpose of this command is to be destructive to the history. why would we want to avoid it?
is there anything inherently wrong with using `force`?

Reply:
Nothing wrong was looking for alternatives, we get this in, for now, thanks.

Reply:
alright 👍 

as usual, thanx for the merge, i really love (contributing to) this project ❤️